lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -719,6 +721,8 @@ def install_puppet_from_freebsd_ports_on( hosts, opts )
           block_on hosts do |host|
             if host['platform'] =~ /freebsd-9/
               host.install_package("puppet")
+            elsif host['platform'] =~ /freebsd-1[234]/
+              host.install_package("sysutils/puppet7")
```

Comment:
`sysutils/puppet` is old puppet 3, and obsoleted and doesn't exist as a pkg for recent FreeBSDs. we could just default to `sysutils/puppet7` on FreeBSD, yanking all that cruft.

Reply:
We already have a backwards-incompatible change merged. maybe just match on `/freebsd/` and drop the freebsd 9 support?